packages/components/src/components/Tooltip/Tooltip.test.tsx

```diff
@@ -0,0 +1,36 @@
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import React from "react";
+import { Default } from "./Tooltip.stories";
```

Comment:
Maybe using a descriptive name would be better to understand the tests :nail_care: 

```suggestion
import { Default as TooltipContainer } from "./Tooltip.stories";
```